book-2-mvc/chapters/ASPNET_INTRO.md

```diff
@@ -77,63 +89,136 @@ public SqlConnection Connection
 }
 ```
 
-![](./images/pTuQO4sdAE.gif)
+### The power of ASP<span>.NET</span> Controllers
```

Comment:
I think we should move this below the "Getting A single walker" section and expand it to go deeper into routing.